pkg/cloud/gcp/actuators/services/compute/computeservice.go

```diff
@@ -11,6 +10,8 @@ import (
 type GCPComputeService interface {
 	InstancesInsert(project string, zone string, instance *compute.Instance) (*compute.Operation, error)
 	ZoneOperationsGet(project string, zone string, operation string) (*compute.Operation, error)
+	InstancesGet(project string, zone string, instance string) (*compute.Instance, error)
```

Comment:
I think this should be singular. We seem to only return one instance.

Reply:
@frobware it's a little confusing, but the section of the api is called "Instances" and it maps to computerservice.Instances.Get(<an instance>) and I think we should keep it as-is for that reason.